release/bin/start.sh

```diff
@@ -72,24 +72,34 @@ case $OP in
         fi
 
         if [ "$COMPONENT" != "taskmanager" ]; then
-            ./bin/openmldb --flagfile=./conf/"$COMPONENT".flags --enable_status_service=true >>  "$LOG_DIR"."$COMPONENT".log 2>&1 &
+            ./bin/openmldb --flagfile=./conf/"$COMPONENT".flags --enable_status_service=true >> "$LOG_DIR"."$COMPONENT".log 2>&1 &
```

Comment:
If openmldb is incompatible, e.g. released bin runs on ubuntu22, will the error msg write to this log file?
If start failed, we'd better to print hint, like 'check log in "$LOG_DIR"."$COMPONENT".log and $LOG_DIR"/"$COMPONENT".INFO'